onnxruntime/contrib_ops/cuda/bert/add_bias_transpose.cu

```diff
@@ -232,24 +283,24 @@ void LaunchAddBiasTranspose(
     cudaStream_t stream, const int num_matrices, const int format, const int max_threads_per_block,
     const int batch_size, const int sequence_length, const int num_heads, const int head_size,
     const half* input, const half* biases, half* output,
-    bool enable_half4) {
+    bool enable_half4, const int* qkv_head_sizes) {
```

Comment:
To be more clear, int head_size shall be renamed as qk_head_size, and const int* qkv_head_sizes changed to int k_head_size.